src/test/java/net/sf/jabref/exporter/layout/format/WrapContentTest.java

```diff
@@ -0,0 +1,57 @@
+package net.sf.jabref.exporter.layout.format;
+
+import net.sf.jabref.exporter.layout.ParamLayoutFormatter;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class WrapContentTest {
+
+    @Test
+    public void testSimpleText() {
+        ParamLayoutFormatter a = new WrapContent();
+        a.setArgument("Bob,Ben");
```

Comment:
Could you please use more real-world examples as arguments, like " < , > ". Makes the tests more readable.